recipes/db.rb

```diff
@@ -1,5 +1,11 @@
 private_ip = my_private_ip()
 fqdn = node['fqdn']
+
+if node['install']['localhost'] == "true"
```

Comment:
@jimdowling: I cannot request changes here as I opened the PR for you. 
In the rest of the PRs you use `node['install']['localhost'].casecmp?("true")` could you please be consistent and use the same here?

Reply:
done.